tests/etl/google_sheet_to_athena.py

```diff
@@ -23,7 +23,7 @@ def integration_test_should__load_sheet_to_athena__when_using_sheetUtil(self):
 
         GoogleSheetToAthena(GoogleSheetsToAthenaSettings(
             source_workbook_url='https://docs.google.com/spreadsheets/d'
-                                '/1W1vICc6wQjumMr9QXNK0bneQCzKFsHacKyrzCBLfsQM/edit?usp=sharing',
+                                '/1W1vICFsHacKyrzCBLfsQM/edit?usp=sharing',
```

Comment:
replace with a generic link that does not point to an internal sheet

Reply:
This is not a true link